ajax/libs/choices.js/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "choices.js",
+  "version": "2.8.6",
+  "description": "A vanilla JS customisable text input/select box plugin",
+  "filename": "choices.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jshjohnson/Choices.git"
+  },
+  "author": "Josh Johnson",
+  "keywords": [
+    "customisable",
+    "input",
+    "select",
+    "vanilla",
+    "plugin",
+    "js"
+  ],
+  "license": "MIT",
+  "homepage": "https://joshuajohnson.co.uk/Choices/",
+  "npmName": "choices.js",
+  "npmFileMap": [
+    {
+      "basePath": "assets/scripts/dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "assets/styles/css",
+      "files": [
+        "**/*"
```

Comment:
And wondering if `**/*` can be used here.
If the answer is no. It will be `styles/css/+(base|choices)*.css`, which can be combined with the below basePath.